R/UK.R

```diff
@@ -273,6 +278,12 @@ UK <- R6::R6Class("UK", # rename to country name
     release_date = NA,
     #' @field resolution The resolution of the data to return
     resolution = "utla",
+    #' @field nhs_raw Raw NHS region data
+    nhs_raw = NA,
+    #' @field authority_data The raw data for creating authority lookup tables
+    authority_data = NA,
+    #' @field nhs_base_url Base url for nhs region data
+    nhs_base_url = "https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/", # nolint
```

Comment:
I like Kath's idea of making this a "meaningful" link despite it adding more URL combining to the code.